### DISCUSSION TITLE
Cracked premium usernames.

### DISCUSSION THREAD

**Discussion by GazpaNova** [games647/FastLogin#593] [Q&A] [answered]
This has probably been asked a ton of times already but here I go

On the spigot page, on the FAQ, it is stated that cracked users can join the server using premium usernames, but I didn't understand how to enable this. I read something about a /premium command: I have autoregister enabled so first-time joining premium users don't have to use /premium.

Do I have to disable it to allow for cracked premium usernames? Thanks in advance.

**Comment by TuxCoding** [comment, marked as answer]
That's a downside of autoregister, it assumes premium for new players. You can however enable `secondAttempt`. Instead you can also disable autoregister and use it as opt-in using the premium command for people that want to autologin.
